src/main/java/de/teamlapen/vampirism/player/vampire/VampirePlayer.java

```diff
@@ -144,6 +168,10 @@ public NBTTagCompound serializeNBT() {
     private IVampireVision activatedVision = null;
     private Method reflectionMethodSetSize = null;
 
+    private int victim = -1;
```

Comment:
probably best to prefix this variable with `feed_` or `bite_` to make it more distinct and adjust the other ones accordingly. VampirePlayer is quite a big class